airbyte-integrations/connectors/source-news-api/source_news_api/news_api.yaml

```diff
@@ -0,0 +1,68 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["articles"]
+  requester:
+    url_base: "https://newsapi.org/v2"
+    http_method: "GET"
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['api_key'] }}"
+    request_options_provider:
+      request_parameters:
+        q: "{{ config['search_query'] }}"
+        searchIn: "{{ ','.join(config['search_in']) }}"
+        sources: "{{ ','.join(config['sources']) }}"
+        domains: "{{ ','.join(config['domains']) }}"
+        excludeDomains: "{{ ','.join(config['exclude_domains']) }}"
+        from: "{{ config['from_date'] }}"
+        to: "{{ config['to_date'] }}"
+        language: "{{ config['language'] }}"
+        sortBy: "{{ config['sort_by'] }}"
+        country: "{{ config['country'] if options['use_country'] is defined else None }}"
```

Comment:
```suggestion
        country: "{{ config.get('country') }}"
```
does the same

Reply:
I disagree. `config.get('country')` returns `None` if `country` is not defined on the `config` object. This returns `None` if `use_config` is not defined on the `options` object. This is because the API doesn't accept these parameters if they're not required. Will add a comment explaining.

Reply:
Oh I see, I read the code wrongly =)